src/System.Windows.Forms/src/System/Windows/Forms/AxHost.cs

```diff
@@ -3576,7 +3576,7 @@ private void DetachWindow()
         private void InformOfNewHandle()
         {
             Debug.Assert(IsHandleCreated, "we got to have a handle to be here...");
-            wndprocAddr = UnsafeNativeMethods.GetWindowLong(new HandleRef(this, Handle), NativeMethods.GWL_WNDPROC);
+            wndprocAddr = User32.GetWindowLong(new HandleRef(this, Handle), User32.GWL.WNDPROC);
```

Comment:
Can this be?
```suggestion
            wndprocAddr = User32.GetWindowLong(this, User32.GWL.WNDPROC);
```

The same applies to other instances